### ISSUE TITLE
[DOXIASITETOOLS-295] Rename o.a.m.doxia.siterenderer.Renderer to o.a.m.doxia.siterenderer.SiteRenderer

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-doxia-sitetools#260]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** opened **[DOXIASITETOOLS-295](https://issues.apache.org/jira/browse/DOXIASITETOOLS-295?redirect=false)** and commented

Class file says "TODO rename to SiteRenderer". The name `Renderer` is too generic.

---

**Affects:** 2.0.0-M5

**Issue Links:**
- [DOXIASITETOOLS-127](https://issues.apache.org/jira/browse/DOXIASITETOOLS-127) rename Renderer (with RendererException) interface to SiteRenderer
 (_**"supercedes"**_)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Affected files:

```
D:\Entwicklung\Projekte\maven-sources\plugins\core\maven-site-plugin\src\main\java\org\apache\maven\plugins\site\render\AbstractSiteRenderingMojo.java
D:\Entwicklung\Projekte\maven-sources\plugins\core\maven-site-plugin\src\main\java\org\apache\maven\plugins\site\render\CategorySummaryDocumentRenderer.java
D:\Entwicklung\Projekte\maven-sources\plugins\core\maven-site-plugin\src\main\java\org\apache\maven\plugins\site\render\ReportDocumentRenderer.java
D:\Entwicklung\Projekte\maven-sources\plugins\core\maven-site-plugin\src\main\java\org\apache\maven\plugins\site\run\DoxiaFilter.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\jxr\maven-jxr-plugin\src\main\java\org\apache\maven\plugin\jxr\AbstractJxrReport.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\maven-changelog-plugin\src\main\java\org\apache\maven\plugin\changelog\ChangeLogReport.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\maven-changes-plugin\src\main\java\org\apache\maven\plugins\changes\AbstractChangesReport.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\maven-pmd-plugin\src\main\java\org\apache\maven\plugins\pmd\AbstractPmdReport.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\maven-pmd-plugin\src\main\java\org\apache\maven\plugins\pmd\exec\PmdExecutor.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\maven-pmd-plugin\src\main\java\org\apache\maven\plugins\pmd\PmdReport.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\maven-project-info-reports-plugin\src\main\java\org\apache\maven\report\projectinfo\AbstractProjectInfoReport.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\maven-project-info-reports-plugin\src\main\java\org\apache\maven\report\projectinfo\dependencies\renderer\DependenciesRenderer.java
D:\Entwicklung\Projekte\maven-sources\plugins\reporting\maven-project-info-reports-plugin\src\main\java\org\apache\maven\report\projectinfo\dependencies\renderer\DependencyManagementRenderer.java
D:\Entwicklung\Projekte\maven-sources\plugins\tools\maven-dependency-plugin\src\main\java\org\apache\maven\plugins\dependency\analyze\AnalyzeReportMojo.java
D:\Entwicklung\Projekte\maven-sources\plugins\tools\maven-invoker-plugin\src\main\java\org\apache\maven\plugins\invoker\InvokerReport.java
D:\Entwicklung\Projekte\maven-sources\plugins\tools\maven-pdf-plugin\src\main\java\org\apache\maven\plugins\pdf\PdfMojo.java
D:\Entwicklung\Projekte\maven-sources\plugins\tools\plugin-tools\maven-plugin-plugin\src\main\java\org\apache\maven\plugin\plugin\PluginReport.java
D:\Entwicklung\Projekte\maven-sources\shared\reporting-impl\src\main\java\org\apache\maven\reporting\AbstractMavenReport.java
```

UT and IT code excluded.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [e94703e1582f3f3693117b14459221a450696012](https://gitbox.apache.org/repos/asf?p=maven-doxia-sitetools.git&a=commit&h=e94703e1582f3f3693117b14459221a450696012).
